backend/routes/post.py

```
@@ -323,25 +308,29 @@ async def create_event_post(
-            # Create the full file path for saving the image
-            file_path = os.path.join(upload_folder, file_name)
-
-            # Ensure the file path is safe and within the intended directory
-            file_path = ensure_safe_path(upload_folder, file_path)
+            # Validate file extension
+            ALLOWED_EXTENSIONS = {".jpg", ".jpeg", ".png", ".gif", ".webp"}
+            file_extension = Path(event_image.filename).suffix.lower()
+            if file_extension not in ALLOWED_EXTENSIONS:
+                raise HTTPException(
+                    status_code=400,
+                    detail="Invalid file type"
```

Ruff (0.8.2)

Trailing comma missing

Add trailing comma

(COM812)